openapi/anchor-platform/bundled-callbacks.yaml

```diff
@@ -1064,8 +1090,8 @@ components:
           $ref: '#/components/schemas/Amount'
         amount_out:
           $ref: '#/components/schemas/Amount'
-        amount_fee:
-          $ref: '#/components/schemas/Amount'
+        fee_details:
+          $ref: '#/components/schemas/FeeDetails'
```

Comment:
Did we remove `amount_fee` from the response we return? I think it should still be there to avoid breaking wallets that still use it. It should just always equal `fee.total`

Reply:
Right `amount_fee` is removed in 3.0 https://stellarorg.atlassian.net/browse/ANCHOR-636 as breaking change